content/handbook/engineering/devops/create/import/_index.md

```
@@ -1,205 +1,147 @@
+このグループは、GitLabインスタンス間および他のプロバイダーからのマイグレーションを行うことで製品をサポートしています。
 
-このページは Import グループに固有のプロセスと情報を扱います。
-[グループ方針ページ](https://about.gitlab.com/direction/create/import/)と
-[カテゴリーごとにサポートする機能](/handbook/product/categories/features/#import-and-integrate)も併せて参照してください。
+このページではImportグループ固有のプロセスと情報をカバーします。[グループ方向性ページ](https://about.gitlab.com/direction/create/import/)および[カテゴリーごとにサポートする機能](/handbook/product/categories/#import-group)も参照してください。
 
 ## 連絡方法
 
-Import グループに連絡を取るには、関連するプロジェクト（通常は [GitLab](https://gitlab.com/gitlab-org/gitlab)）に Issue を作成し、`~"group::import"` ラベルとその他の[適切なラベル](#issue-labels)を付与するのが最善です。
-そのうえで、関連するプロダクトマネージャーやエンジニアリングマネージャーに気軽に ping してください。
+Importグループに連絡するには、[GitLabプロジェクト](https://gitlab.com/gitlab-org/gitlab)でIssueを作成し、`~"group::import"`ラベルとその他の[適切なラベル](#issue-labels)を追加するのが最善です。その後、関連するプロダクトマネージャーおよび/またはエンジニアリングマネージャーにpingしてください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)